### PR TITLE
Login Review

### PR DESCRIPTION
**Pull request by akhilalla94**

### CODE REVIEW

File: application.py [hunk 1]
```diff
@@ -18,33 +18,75 @@
...
+                return render_template("registration.html", message = "Please enter correct username/password")
+        else:
+
+            return redirect(url_for('index'))
+    else:
+
+        return "<h1>Please login/register instead</h1>"
+
+
 @app.route("/userDetails",methods=["POST","GET"])
```
Comment (ashishk144) [inline application.py]: As said earlier change this URI to /register and delete the /register written above

File: application.py [hunk 2]
```diff
@@ -18,33 +18,75 @@
 app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 db.init_app(app)
 
-
 with app.app_context():
     db.create_all()
```
Comment (ashishk144) [inline application.py]: create_all is a function to create tables, there is no need to run the method each time the flask app is run

**Review by ashishk144 — COMMENTED** [review]
You can merge after resolving the comments.

File: application.py (hunk 1 above)
Comment (akhilalla94) [reply application.py]: K bro I will make changes based on the feedback

**Pull request merged**